packages/codelabs/src/basics/web-components.md

```diff
@@ -4,34 +4,34 @@
 
 In this codelab, you will learn about the basics of web components and how they work.
 
-Web components are a set of low-level browser APIs that allow us to write modular, encapsulated and reusable HTML elements. Web components are based on web standards and work in any environment that supports basic HTML and JavaScript. This means that there is no complex setup required in order for you to get started.
+Web components are a set of low-level browser features that allow us to write modular, encapsulated and reusable HTML elements. Web components are based on web standards and work in any environment that supports basic HTML and JavaScript. This means that there is no complex setup required for you to get started.
 
-The APIs of web components align well with the way that browsers work and have always worked. This means that they are pretty low level and straightforward. For most projects, libraries or frameworks will still be used, but instead of each framework developing their own component model, they can preferably use the components that are baked into the browser.
+The features of web components align well with the way that browsers work and have always worked. This means that they are pretty low level and straightforward. For most projects, libraries or frameworks will still be used, but instead of each framework developing their own component model, they can preferably use the component technology that is baked into the browser.
```

Comment:
```suggestion
Web components align with the way that browsers have always worked, they are pretty low level and straightforward. For most projects you will still want to use libraries or frameworks. But instead of each framework developing their own component model, they can use the features that are already baked into the browser.
```